node/src/tests/test-WebRtcTransport.ts

```diff
@@ -306,6 +311,7 @@ test('webRtcTransport.connect() with wrong arguments rejects with TypeError', as
 		fingerprints :
 		[
 			{
+				// @ts-ignore.
 				algorithm : 'sha-256000',
```

Comment:
> Now that we have types, does it make any sense to test with wrong values that the typing system would not allow to happen?

Yes, because we cannot assume that every user uses TypeScript or validates network received data no matter it's created by media soup-client or not.

Reply:
In Rust you cannot pass an invalid enum or type, in JS you can.

Reply:
> In Rust you cannot pass an invalid enum or type, in JS you can.

>> or validates network received data no matter it's created by media soup-client or not.

Such argument applies to rust too.

Reply:
In Rust you **cannot** pass a random string.

Reply:
> In Rust you cannot pass a random string.

I'm talking about runtime. We can't control what's coming from the network.

Reply:
I don't care what we receive from the network in a Rust app. Whatever we receive from the network we have to convert into a proper `webrtc_transport_options` Rust type **first** before we pass it to `router.create_webrtc_transport()`.

Reply:
This is not being done. I'm removing the `skip` from the test and properly throw when receiving wrong data.